operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -56,7 +56,7 @@ kiali_defaults:
     jaeger:
       jaeger_service: "jaeger-query"
       service: "tracing"
-      service_namespace: ""
+      namespace: ""
```

Comment:
please alphabetize - all settings are alphanumerically sorted.

Reply:
the "tracing" setting should move below "prometheus" to maintain alphabetical sort order of the settings.

```
prometheus:
  custom_metrics_url: ""
  url: ""
tracing:
  namespace: ""
  service: ""
  url: ""